x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/tour.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiSpacer, EuiTourStep } from '@elastic/eui';
+import React, { useCallback, useEffect, useState } from 'react';
+import { NEW_TERMS_TOUR_ACTIVE_KEY } from '../../../../../common/constants';
+import * as i18n from './translations';
+
+export interface Props {
+  children: React.ReactElement;
+}
+
+export const RulesPageTourComponent: React.FC<Props> = ({ children }) => {
+  const tourConfig = {
+    currentTourStep: 1,
+    isTourActive: true,
+    tourPopoverWidth: 300,
+  };
+
+  const [tourState, setTourState] = useState(() => {
+    const tourStateString = localStorage.getItem(NEW_TERMS_TOUR_ACTIVE_KEY);
+
```

Comment:
shoudn't we use `useKibana().services` for accessing localStorage? It handles serialization automatically